neo/app/assets/javascripts/common/services.js

```diff
@@ -0,0 +1,69 @@
+angular.module('cloudberry.common', [])
+  .service('Asterix', function($http, $timeout) {
```

Comment:
No clear either... my understanding is service return a function and factory return a object which contains functions. But I think both works ok for our application